include/trackcpp/tracking.h

```diff
@@ -79,6 +80,23 @@ Status::type track_elementpass (
 
 }
 
+template <typename T>
```

Comment:
use of same name functions that differ in signature by their argument types... nice!
hail c++!